aten/src/ATen/native/LinearAlgebra.cpp

```diff
@@ -8,50 +8,100 @@
 namespace at {
 namespace native {
 
-// For backward, we save svd.
+// This method calculates svd with an additional constraint (det(UV) = +1),
+// which will be used in det methods backward.
+//
 // http://www.ics.forth.gr/cvrl/publications/conferences/2000_eccv_SVD_jacobian.pdf
-// But instead of gesvd SVD A = U(A) Sig(A) V(A)^T, which doesn't specify signs
+// Instead of gesvd SVD A = U(A) Sig(A) V(A)^T, which doesn't specify signs
```

Comment:
What do you mean by "doesn't specify signs?"

Reply:
signs of determinants of U and V. The sentence continues on the next line :). No worries. I'll ask someone with more experience to review this.